File: Java/app/src/main/java/dev/ogabek/java/MainActivity.java
package dev.ogabek.java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private Button signUpBtn;
    private EditText et_username, et_password;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();

    }

    private void initViews() {
        signUpBtn = findViewById(R.id.btn_sign_up);
        et_username = findViewById(R.id.et_username);
        et_password = findViewById(R.id.et_password);
        String text = "Username :  " + et_username + "\n" + "Password :  " + et_password;
        signUpBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(), text, Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/MainActivity.kt
package dev.ogabek.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    private lateinit var signUpBtn: Button
    private lateinit var et_username: EditText
    private lateinit var et_password:EditText

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initViews()
    }

    private fun initViews() {
        signUpBtn = findViewById(R.id.btn_sign_up)
        et_username = findViewById(R.id.et_username)
        et_password = findViewById(R.id.et_password)
        val text = "Username :  $et_username\nPassword :  $et_password"
        signUpBtn.setOnClickListener {
            Toast.makeText(applicationContext, text, Toast.LENGTH_SHORT).show()
        }
    }
}